specification/apicenter/DataApi/models/api.tsp

```diff
@@ -0,0 +1,120 @@
+namespace ApiCenter.DataApi.Models;
+
+@doc("API contract")
+model Api {
+  @doc("The resource name.")
+  @maxLength(50)
+  name: string;
+
+  @doc("The name of the API.")
+  @maxLength(50)
+  title: string;
+
+  @doc("Short descrption of the API.")
+  @maxLength(200)
+  summary?: string;
+
+  @doc("The description of the API")
+  @maxLength(1000)
+  description?: string;
+
+  @doc("Kind of API. For example, REST or GraphQL.")
+  kind: string;
```

Comment:
This should be an enum. How would customers know what values could be passed? Even as docs, you say, "for example, REST or GraphQL". Are those values - which are supposed to be case-sensitive - cased that way? Are there others, since you say, "for example". Use enums for `kind` typically.

Reply:
Fixed in latest revision.